cyberdrop_dl/config_definitions/config_settings.py

```diff
@@ -44,6 +44,8 @@ class Logs(AliasModel):
     download_error_urls_filename: NonEmptyStr = "Download_Error_URLs.csv"
     scrape_error_urls_filename: NonEmptyStr = "Scrape_Error_URLs.csv"
     rotate_logs: bool = False
+    seperate_folders: bool = False
```

Comment:
Instead of adding a new option, change the existing log file name options to take a `path`. If the user wants to use a subfolder, they can use something like `Download_Errors_Logs/Download_Error.csv`. This will also allow them to save all the logs of all the configs on the same folder by using an absolute path instead of a relative path, but they will have to also set  `rotate_logs` to make sure the do not override each other